src/modules/esl-carousel/plugin/touch/esl-carousel.touch.mixin.ts

```diff
@@ -90,11 +115,32 @@ export class ESLCarouselTouchMixin extends ESLCarouselPlugin {
     if (this.$$attr('dragging', false) !== null) {
       event.preventDefault();
       const point = getTouchPoint(event);
-      const offset = point.x - this.startPoint.x;
+      const offset = this.$host.config.vertical ? point.y - this.startPoint.y : point.x - this.startPoint.x;
       // ignore single click
       offset !== 0 && this.$host.renderer.commit(offset);
```

Comment:
This piece of code is also duplicated above for `_onPointerMove,` maybe it makes sense to keep it separately